titan/chain.json

```diff
@@ -0,0 +1,169 @@
+    },
+    "versions": [
+      {
+        "name": "v2.0.1",
+        "tag": "v2.0.1",
+        "height": 0,
+        "recommended_version": "v2.0.1",
+        "compatible_versions": [
+          "v2.0.1"
+        ],
+        "cosmos_sdk_version": "0.47.6-titan.4",
+        "consensus": {
+          "type": "cometbft",
+          "version": "0.37.4"
+        },
+        "ibc_go_version": "7.3.0",
+        "cosmwasm_version": "0.45.0",
+        "cosmwasm_enabled": true,
+        "ics_enabled": [
+          "ics20-1",
+          "ics27-1"
+        ]
+      }
+    ]
+  },
+  "images": [
+    {
+      "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/titan/images/chain.png",
+      "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/titan/images/chain.svg"
+    }
+  ],
+  "peers": {
+    "seeds": [
+      {
+        "id": "cf2f46da018e9b61c2db74012bd930d292478bb6",
+        "address": "titan-p2p-1.titanlab.io:26656",
+        "provider": "Titanlab.io"
+      },
+      {
+        "id": "0538c914eccc67a335eb64d99406c71ba7b110ca",
+        "address": "titan-p2p-2.titanlab.io:26656",
+        "provider": "Titanlab.io"
+      },
+      {
+        "id": "bee5ef5680cf90fe40d6cde872cdc52e53c8338d",
+        "address": "titan-p2p-seed-1.titanlab.io:26656",
+        "provider": "Titanlab.io"
+      },
+      {
+        "id": "dbc0da89a67a9ae8924f2b125e6ffa0043e1fe25",
+        "address": "titan-p2p-full-1.titanlab.io:26656",
+        "provider": "Titanlab.io"
+      }
+    ],
+    "persistent_peers": [
+      {
+        "id": "cf2f46da018e9b61c2db74012bd930d292478bb6",
+        "address": "titan-p2p-1.titanlab.io:26656",
+        "provider": "Titanlab.io"
+      },
+      {
+        "id": "0538c914eccc67a335eb64d99406c71ba7b110ca",
+        "address": "titan-p2p-2.titanlab.io:26656",
+        "provider": "Titanlab.io"
+      }
+    ]
+  },
+  "apis": {
+    "rpc": [
+      {
+        "address": "https://titan-rpc.titanlab.io:443",
+        "provider": "Titanlab.io"
+      },
+      {
+        "address": "https://titan-rpc-1.titanlab.io:443",
+        "provider": "Titanlab.io"
+      },
+      {
+        "address": "https://titan-rpc-2.titanlab.io:443",
+        "provider": "Titanlab.io"
+      },
+      {
+        "address": "https://titan-rpc-seed-1.titanlab.io:443",
+        "provider": "Titanlab.io"
+      },
+      {
+        "address": "https://titan-rpc-full-1.titanlab.io:443",
+        "provider": "Titanlab.io"
+      }
+    ],
+    "rest": [
+      {
+        "address": "https://titan-lcd.titanlab.io:443",
+        "provider": "Titanlab.io"
+      }
+    ],
+    "grpc": [],
+    "evm-http-jsonrpc": [
+      {
+        "address": "https://titan-json-rpc.titanlab.io:443",
+        "provider": "Titanlab.io"
+      }
+    ]
+  },
+  "explorers": [
+    {
+      "kind": "ping.pub",
+      "url": "https://titan-explorer-light.titanlab.io",
```

Comment:
Domain not in DNS?

Reply:
Thank you for review. That website is already online. Please help me review again.